proc/proc_test.go

```
@@ -778,7 +778,9 @@ func TestStacktraceGoroutine(t *testing.T) {
 
 		for i, g := range gs {
 			locations, err := g.Stacktrace(40)
-			assertNoError(err, t, "GoroutineStacktrace()")
+			if err != nil {
```

This needs a comment / explanation / TODO about why we're ignoring errors.